src/commands/cmd_tdigest.cc

```diff
@@ -131,6 +131,44 @@ class CommandTDigestInfo : public Commander {
   std::string key_name_;
 };
 
+class CommandTDigestAdd : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    key_name_ = args[1];
+    values_.reserve(args.size());
+
+    for (size_t i = 2; i < args.size(); i++) {
+      auto value = ParseFloat<>(args[i]);
```

Comment:
```suggestion
      auto value = ParseFloat(args[i]);
```

I prefer to remove this `<>`.